util/bpmath/types.go

```diff
@@ -0,0 +1,19 @@
+package bpmath
+
+import (
+	"cosmossdk.io/math"
+)
+
+type Rounding uint
+
+const (
+	DOWN = iota
```

Comment:
```suggestion
	DOWN Rounding = iota
```
keeps the constants defined under a type